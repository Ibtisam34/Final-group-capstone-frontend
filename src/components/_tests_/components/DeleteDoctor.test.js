import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../../../redux/configureStore';
import Removedoctor from '../../../components/AddDoctor/DeleteDoctor';

describe('Delete Doctor', () => {
  it('renders correctly', () => {
    window.localStorage.setItem('userId', 1);
    const tree = render(<Provider store={store}><Router><Removedoctor /></Router></Provider>);
    expect(tree).toMatchSnapshot();
  });
});