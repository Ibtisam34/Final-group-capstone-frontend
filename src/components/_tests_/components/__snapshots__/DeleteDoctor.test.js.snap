// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Doctor renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="reservations-container"
        style="background-position: center; background-size: cover; padding: 2rem;"
      >
        <div
          class="reservation-header-links"
          style="position: fixed; top: 0px;"
        >
          <a
            href="/"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              style="color: rgb(65, 70, 75); font-size: 2rem; margin: 5px;"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
              />
            </svg>
          </a>
        </div>
        <form
          class="form"
          style="display: flex; align-items: center; flex-direction: column;"
        >
          <h1>
            Remove a doctor from our Listing
          </h1>
          <select
            id="selectDropdown"
            placeholder="Choose a doctor you want to reserve."
            style="outline: none; width: 95%; border-radius: 2px; margin: 2rem 0px;"
          >
            <option
              disabled=""
            >
              Choose a doctor
            </option>
          </select>
          <button
            class="submit book-btn"
            to="/"
            type="submit"
          >
            Submit
          </button>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="reservations-container"
      style="background-position: center; background-size: cover; padding: 2rem;"
    >
      <div
        class="reservation-header-links"
        style="position: fixed; top: 0px;"
      >
        <a
          href="/"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            style="color: rgb(65, 70, 75); font-size: 2rem; margin: 5px;"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
            />
          </svg>
        </a>
      </div>
      <form
        class="form"
        style="display: flex; align-items: center; flex-direction: column;"
      >
        <h1>
          Remove a doctor from our Listing
        </h1>
        <select
          id="selectDropdown"
          placeholder="Choose a doctor you want to reserve."
          style="outline: none; width: 95%; border-radius: 2px; margin: 2rem 0px;"
        >
          <option
            disabled=""
          >
            Choose a doctor
          </option>
        </select>
        <button
          class="submit book-btn"
          to="/"
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
